cookbooks/travis_build_environment/recipes/redis.rb

```diff
@@ -1,9 +1,9 @@
-apt_repository 'rwky-redis' do
-  uri 'http://ppa.launchpad.net/rwky/redis/ubuntu'
+apt_repository 'chris-lea-redis-server' do
+  uri 'http://ppa.launchpad.net/chris-lea/redis-server/ubuntu'
   distribution node['lsb']['codename']
   components ['main']
-  key '5862E31D'
-  keyserver 'hkp://ha.pool.sks-keyservers.net'
+  key 'C7917B12'
+  keyserver 'hkp://keyserver.ubuntu.com'
```

Comment:
This key should be on `ha.pool.sks-keyservers.net`, too, which is generally much more reliable that the ubuntu keyserver.  Was it failing for you?